src/main/blackbox/blackbox_io.c

```diff
@@ -470,7 +470,7 @@ bool blackboxDeviceOpen(void)
      *
      * 9 / 1250 = 7200 / 1000000
      */
-    blackboxWriteChunkSize = MAX((masterConfig.looptime * 9) / 1250, 4);
+    blackboxWriteChunkSize = MAX((GUARD_LOOPTIME * 9) / 1250, 4);
```

Comment:
This is called only on init and used over multiple loops ... if getGyroSampleDivider() has part that returns desired looptime, it would work great here (gyro implementation must round up anyway)